otcextensions/tests/functional/sdk/mrs/v1/test_host.py

```diff
@@ -0,0 +1,60 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+import openstack
+
+from otcextensions.tests.functional import base
+
+_logger = openstack._log.setup_logging('openstack')
+
+
+class TestHost(base.BaseFunctionalTest):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Please do not use setUpClass. SDK is now forbidding that and I will go through the remaining code shortly to remove those